wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/WireExtension.kt

```diff
@@ -207,24 +207,36 @@ open class WireExtension(
     return result
   }
 
+  /**
+   * Defines a Java target. See [com.squareup.wire.schema.JavaTarget]
```

Comment:
Comments should be sentences!

Reply:
déjà vu !!!

Thank you, all addressed.